src/errors/rusty-bot-errors.ts

```diff
@@ -0,0 +1,31 @@
+export class MissingEnvironmentVariableError extends Error {
+  constructor(variableName: string) {
+    super(`${variableName} must be provided`);
+    this.name = 'MissingEnvironmentVariableError';
+  }
+}
+
+export class InvalidColorStringError extends Error {
+  constructor(colorString: string) {
+    super(`${colorString} is invalid`);
+    this.name = 'InvalidColorStringError';
+  }
+}
+
+export class RustyBotCommandError extends Error {
+  constructor(replyMessage: string, internalMessage?: string) {
+    super(replyMessage);
+    this.name = 'RustyBotCommandError';
+
+    if (internalMessage) console.error(internalMessage);
```

Comment:
Does our linter/prettier prefer this to using brackets on single line if statements? I just don't see them super often. I think I personally prefer always having bracketed if statements but I don't feel super strongly about that.

Reply:
I agree with preferring brackets always. I say that as someone that will write if statements in their own projects though so I definitely don't feel strongly. I have just seen a lot of feedback elsewhere on avoiding this syntax.